examples/chip-tool/templates/tests.js

```diff
@@ -148,13 +149,14 @@ function getTests()
 
   const ThreadNetworkDiagnostics = [
     'Test_TC_DIAGTH_1_1',
-    'Test_TC_DIAGSW_3_2',
   ];
 
   const WindowCovering = [
     'Test_TC_WNCV_1_1',
     'Test_TC_WNCV_2_1',
     'Test_TC_WNCV_2_2',
+    'Test_TC_WNCV_2_3',
```

Comment:
Please remove 2.3 , i ll take care of it , it is far more complex and the WindowCovering is nor ready nor spec compliant